src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -177,20 +177,20 @@ int nextIdx( inout idxs_t idxs ) {
   idxs = idxs >> 2;
   return int( tmp.x + tmp.y + tmp.z + tmp.w );
 }
-#else // !HAVE_EXT_texture_integer
+#else // !HAVE_EXT_texture_integer || !r_highPrecisionRendering
 const int lightsPerLayer = 4;
-uniform sampler3D u_LightTiles;
+uniform usampler3D u_LightTilesInt;
```

Comment:
This sampler should stay as `sampler3D u_LightTiles`. The reason for the distinction is so that `GLShaderManager::ShaderPostProcess()` can work correctly without having to process macros in shaders, which is what we used previously to select the correct sampler type.

Reply:
Why do `u_LightTilesInt` is passed as argument in the other use case? It looks to never be modified.

Can it be used as a global variable instead? This would prevent us to duplicate some lines with the variants (up to 4 variants with `computeDynamicLights`…

Reply:
Both use cases should pass it as argument, I forgot to add it to the non-integer one in #1105.

It can't be global with bindless textures whose handles are sourced from a buffer. We can just ditch the version that uses non-integer textures, since integer textures are supported pretty much everywhere.

Reply:
OK, the current code should now be correct.